astro/src/content/docs/apis/jwt.mdx

```diff
@@ -239,6 +239,11 @@ The refresh token may be provided either in the HTTP request body or as a cookie
   <APIField name="token" type="String" optional>
     The previously issued encoded access token.  When provided on the request, this value will be relayed in the related JWT Refresh webhook event within the `original` field.
   </APIField>
+  <APIField name="timeToLiveInSeconds" type="Integer" optional since="1.99.9">
+    The time to live for the new access token.
+
+    When provided on the request, this value will override the default <InlineField>tenant.jwtConfiguration.timeToLiveInSeconds</InlineField> or <InlineField>application.jwtConfiguration.timeToLiveInSeconds</InlineField> values, but it must be less than or equal to those values.
```

Comment:
chose to not wade into the complexity of what can happen downstream from here (lambda changes things, etc.)